iOS/Flows/Profile/ProfileHeaderView.swift

```diff
@@ -52,22 +52,24 @@ class ProfileHeaderView: BaseView {
     }
     
     @MainActor
-    func configure(with user: User) {
-        
-        self.personView.set(person: user)
-        self.nameLabel.setText(user.givenName)
-        if let position = user.quePosition {
-            self.memberLabel.setText("Member #\(position)")
-        }
-        
-        if user.isCurrentUser {
-            let nowTime = Date.hourMinuteTimeOfDay.string(from: Date())
-            self.timeLabel.setText(nowTime)
-        } else {
-            self.timeLabel.setText(user.getLocalTime())
+    func configure(with person: PersonType) {
```

Comment:
Supports showing information for a CNContact as well.

Reply:
sick